sdk/storage/azure-storage-blob/azure/storage/blob/aio/_lease_async.py

```diff
@@ -3,51 +3,58 @@
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
-# pylint: disable=invalid-overridden-method
 
-from typing import (  # pylint: disable=unused-import
-    Union, Optional, Any, IO, Iterable, AnyStr, Dict, List, Tuple,
-    TypeVar, TYPE_CHECKING
-)
+import uuid
+
+from typing import Any, Optional, Union, TYPE_CHECKING
 
 from azure.core.exceptions import HttpResponseError
 from azure.core.tracing.decorator_async import distributed_trace_async
 
-from .._shared.response_handlers import return_response_headers, process_storage_error
+from .._shared.response_handlers import process_storage_error, return_response_headers
 from .._serialize import get_modify_conditions
-from .._lease import BlobLeaseClient as LeaseClientBase
 
 if TYPE_CHECKING:
+    from azure.storage.blob.aio import BlobClient, ContainerClient
     from datetime import datetime
-    from .._generated.operations import BlobOperations, ContainerOperations
-    BlobClient = TypeVar("BlobClient")
-    ContainerClient = TypeVar("ContainerClient")
 
 
-class BlobLeaseClient(LeaseClientBase):
+class BlobLeaseClient(object):
```

Comment:
Don't need to inherit from `object`. That's a Python 2 thing I think.

Reply:
Does this apply to sync as well or only async? This comment only exists in the async version I think

Reply:
Both